apps/backend/src/modules/routes/route.entity.ts

```diff
@@ -58,12 +65,13 @@ class RouteEntity implements Entity {
 		this.description = description;
 		this.pois = pois;
 		this.createdByUserId = createdByUserId;
+		this.userRoute = userRoute?.[0] ?? null;
```

Comment:
let's name it as saved user route to make it clearer why it's a single item. 
```suggestion
		this.savedUserRoute = userRoute?.[0] ?? null;
```